bsp_diff/caas/vendor/intel/external/onevpl-intel-gpu/0004-Adding-ADL-N-support.patch

```diff
@@ -117,16 +104,22 @@ index 07ff71b..33cc609 100755
  
  };
  
-@@ -455,7 +457,27 @@ typedef struct {
+@@ -455,6 +457,11 @@ typedef struct {
      { 0x46C2, MFX_HW_ADL_P, MFX_GT2 },//ADL-P
      { 0x46C3, MFX_HW_ADL_P, MFX_GT2 },//ADL-P
  
--
 +    /* ADL-N */
 +    { 0x46D0, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
 +    { 0x46D1, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
 +    { 0x46D2, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
-+    
++
+     /*RPL-P*/
+     { 0xA7A0, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
+     { 0xA720, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
+@@ -463,6 +470,22 @@ typedef struct {
+     { 0xA721, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
+     { 0xA7A9, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
```

Comment:
There are duplicates for RPL-P:

```
   /*RPL-P*/
    { 0xA7A0, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA720, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A8, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A1, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA721, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A9, MFX_HW_ADL_P, MFX_GT2 },//RPL-P

    /* ADL-N */
    { 0x46D0, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
    { 0x46D1, MFX_HW_ADL_N, MFX_GT1 },//ADL-N
    { 0x46D2, MFX_HW_ADL_N, MFX_GT1 },//ADL-N

    /*RPL-P*/
    { 0xA7A0, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA720, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A8, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A1, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA721, MFX_HW_ADL_P, MFX_GT2 },//RPL-P
    { 0xA7A9, MFX_HW_ADL_P, MFX_GT2 },//RPL-P

```

It seems like the commits are duplicated:
```

**d12e46e (HEAD) Add device IDs for RPL-P**
2a17924 Adding ADL-N support.
7e9e0c8 Add missing ADL device IDs
**4259734 (github/celadon/s/mr0/stable) Add device IDs for RPL-P**
97b2b58 Add Android T definitions for onevpl-intel-gpu

```

Reply:
Duplicate commit is removed by https://github.com/projectceladon/vendor-intel-utils-vertical-iot/pull/27